plugin/Gui/AdminPageApiSettings.php

```diff
@@ -100,7 +102,15 @@ private function addFormModelAPI()
 		$pFormModel->addInputModel($pInputModelApiKey);
 		$pFormModel->setGroupSlug('onoffice-api');
 		$pFormModel->setPageSlug($this->getPageSlug());
-		$pFormModel->setLabel(__('API settings', 'onoffice-for-wp-websites'));
+
+		$pFieldnames = new Fieldnames(new FieldsCollection());
```

Comment:
Is `$pFieldnames` just here to  check the connection?

Reply:
yes, it check connection